metadataservice/server/engine.py

```diff
@@ -272,7 +272,7 @@ class EventHandler(DefaultHandler):
     def get(self):
         database = self.settings['db']
         query = utils.unpack_params(self)
-        docs = database['event'].find(query)
+        docs = database['event'].find(query, sort=[('time', 1)])
```

Comment:
Sorting on cursor is a performance bottleneck, that's why I have left it out. We can sort locally on the client side